pages/dashboard/views/index.jsx

```diff
@@ -4,20 +4,12 @@ import {
   Link,
   Snippet,
   Header,
-  PanelList
+  PanelList,
+  ExpandingPanel
 } from '@asl/components';
 import TaskList from '@asl/pages/pages/task/list/views/tasklist';
-
-const EstablishmentPanel = ({ id, name }) => (
-  <Fragment>
-    <p>
-      <Link page="establishment.dashboard" establishmentId={id} label={name} />
-    </p>
-    <p>
-      <Link page="profile.invite" establishmentId={id} label={<Snippet>pages.dashboard.invite</Snippet>}/>
-    </p>
-  </Fragment>
-);
+import sortBy from 'lodash/sortBy';
```

Comment:
Sorry to be a pedant, but I much prefer `import`s to be organised - external with external, owned with owned, local with local: eg

```js
import React from 'react';
import { connect } from 'react-redux';
import sortBy from 'lodash/sortBy';
import map from 'lodash/map';

import { Button } from '@ukhomeoffice/react-components';
import { Link, Snippet } from '@asl/components';
import TaskList from '@asl/pages...';

import LocalThing from './path/to/local/thing';
import AnotherLocalThing from './path/to/other/local/thing';
```

It just makes it easier to see what is what at a glance

Reply:
done